asapdiscovery-docking/asapdiscovery/docking/workflows/cross_docking.py

```diff
@@ -168,7 +186,9 @@ def cross_docking_workflow(inputs: CrossDockingWorkflowInputs):
     # using dask here is too memory intensive as each worker needs a copy of all the complexes in memory
     # which are quite large themselves, is only effective for large numbers of ligands and small numbers of complexes
     logger.info("Selecting pairs for docking")
-    selector = PairwiseSelector()
+    # TODO: MCS takes an n_select arg but Pairwise does not...meaning we are losing that option the way this is written
```

Comment:
Pairwise doesn't take an n-select as there is AFAIK no way to conceptually rank the compounds right. To make it modular we could make it a kwarg rather than an arg (with appropriate *args and **kwargs)? And then Pairwise can just ignore the kwarg (I think).

Reply:
can we bump that to a future PR? this works for my purpose now and I'm not sure how to implement what you're suggesting

Reply:
Yep of course